coderd/coderd.go

```diff
@@ -1038,6 +1043,10 @@ func New(options *Options) *API {
 				r.Use(httpmw.ExtractUserParam(options.Database))
 				r.Get("/debug-link", api.userDebugOIDC)
 			})
+			r.Route("/derp", func(r chi.Router) {
+				r.Get("/traffic", options.DERPServer.ServeDebugTraffic)
+			})
+			r.Mount("/expvar", expvar.Handler()) // contains DERP metrics as well as cmdline and memstats
```

Comment:
Sucks that they don't expose prometheus metrics :/

We might have to write a converter from these expvar values to prometheus values

Reply:
Yeah, but I'll save that for a different PR if we find these useful.